### ISSUE TITLE
Docker image, can't connect to postgres

### ISSUE THREAD

**Issue by bodinsamuel** [ankane/pghero#228]
Hi,

I wanted to try the docker image for pgHero but I ran into an issue. 
The docker build can not connect to pg, and I don't know what to do :/

example:
```bash
psql postgres://proxyuser:<redacted>@127.0.0.1:5432/postgres
psql (10.4, server 9.6.6)
Type "help" for help.

postgres=> \q
# connection is working

docker run -ti -e DATABASE_URL=postgres://proxyuser:<redacted>@127.0.0.1:5432/postgres -p 8080:8080 ankane/pghero
[6] Puma starting in cluster mode...
[6] * Version 3.12.0 (ruby 2.5.1-p57), codename: Llamas in Pajamas
[6] * Min threads: 1, max threads: 16
[6] * Environment: production
[6] * Process workers: 3
[6] * Preloading application
[6] * Listening on tcp://0.0.0.0:8080
[6] ! WARNING: Detected 1 Thread(s) started in app boot:
[6] ! #<Thread:0x000055f9960b6f18@/usr/local/bundle/gems/activerecord-5.2.1/lib/active_record/connection_adapters/abstract/connection_pool.rb:297 sleep> - /usr/local/bundle/gems/activerecord-5.2.1/lib/active_record/connection_adapters/abstract/connection_pool.rb:299:in `sleep'
[6] Use Ctrl-C to stop
[6] - Worker 0 (pid: 9) booted, phase: 0
[6] - Worker 1 (pid: 20) booted, phase: 0
[6] - Worker 2 (pid: 31) booted, phase: 0
Started GET "/" for 172.17.0.1 at 2018-09-06 12:40:38 +0000
Processing by PgHero::HomeController#index as HTML
Completed 500 Internal Server Error in 8ms
  
PG::ConnectionBad (could not connect to server: Connection refused
	Is the server running on host "127.0.0.1" and accepting
	TCP/IP connections on port 5432?
):
[....]
```

**Comment by ankane** [comment]
Hey @bodinsamuel, please check out the Contributing Guide for how to get help.

**Comment by devendersangwan855** [comment]
@bodinsamuel 
This is how I  made it work on my Windows 10 Home:
1. I followed the answer to this [StackOverflow question](https://stackoverflow.com/questions/31249112/allow-docker-container-to-connect-to-a-local-host-postgres-database):
2. I used `host.docker.internal` as the hostname in database_url property while running the docker container.
`docker run  -ti -e DATABASE_URL=postgres://user:password@host.docker.internal:5432/your_dbname -p 8080:8080 ankane/pghero`
